--- linux/include/linux/fs.h-dist	2004-10-27 13:11:43.000000000 -0700
+++ linux/include/linux/fs.h	2004-10-27 13:12:00.000000000 -0700
@@ -43,7 +43,7 @@
 #define NR_OPEN (1024*1024)	/* Absolute upper limit on fd num */
 #define INR_OPEN 1024		/* Initial setting for nfile rlimits */
 
-#define BLOCK_SIZE_BITS 10
+#define BLOCK_SIZE_BITS 9
 #define BLOCK_SIZE (1<<BLOCK_SIZE_BITS)
 
 /* And dynamically-tunable limits and defaults: */
