--- linux/kernel/printk.c-dist	2004-10-27 16:53:31.000000000 -0700
+++ linux/kernel/printk.c	2004-10-27 17:00:01.000000000 -0700
@@ -59,7 +59,7 @@
 DECLARE_WAIT_QUEUE_HEAD(log_wait);
 
 int console_printk[4] = {
-	DEFAULT_CONSOLE_LOGLEVEL,	/* console_loglevel */
+	MINIMUM_CONSOLE_LOGLEVEL,	/* console_loglevel */
 	DEFAULT_MESSAGE_LOGLEVEL,	/* default_message_loglevel */
 	MINIMUM_CONSOLE_LOGLEVEL,	/* minimum_console_loglevel */
 	DEFAULT_CONSOLE_LOGLEVEL,	/* default_console_loglevel */
